src/client/components/search/Search.jsx

```diff
@@ -51,6 +51,7 @@ const Search = ({
         required
         onFocus={ focusEndOfInput }
         spellCheck="false"
+        maxLength="15"
```

Comment:
2 things:
1. It works, but wouldn't this usually be done like `maxLength={ 15 }`?
2. If you do use quotes, they should be single quotes

Reply:
changing to 
```javascript
spellCheck={ false }
maxLength={ 15 }
```
worked and is probably better